Wikipedia/Code/WMFSettingsMenuItem.m

```diff
@@ -209,6 +209,19 @@ + (WMFSettingsMenuItem *)itemForType:(WMFSettingsMenuItemType)type {
                                               disclosureText:nil
                                                   isSwitchOn:NO];
         }
+        case WMFSettingsMenuItemType_TemporaryAccount: {
+            NSString *title = WMFLocalizedStringWithDefaultValue(@"settings-temp-account-title", nil, nil, @"Temporary account", @"Temporary account title.");
```

Comment:
You could replace this string with `CommonStrings.tempAccount` to use existing translations